Podfile

```diff
@@ -14,6 +14,7 @@ def externalRegularDependencies
   pod 'BetterSegmentedControl', '~> 2.0.0'
   pod 'TransitionButton', '0.5.3'
   pod 'SkyFloatingLabelTextField', '~> 3.0'
+  pod 'SwiftLint', '~> 0.50', :inhibit_warnings => true
```

Comment:
what's the sense of adding lint warnings and taking them out simultaneously ?

Reply:
I believe by this we inhibit only warnings inside the code of pod (SwiftLint in this case). Guess it not so important but I decided why not. Look what we have in Authenticator
```
def project_pods
    pod 'OneginiSDKiOS', '~> 11.0.0'
    pod 'NVActivityIndicatorView', '5.0.0'
    pod 'KeychainAccess', :inhibit_warnings => true
    pod 'Dip', '7.1.1', :inhibit_warnings => true
    pod 'SwiftLint', '~> 0.50', :inhibit_warnings => true
end

Reply:
I added warnings inhibitions for other 3rd party libraries because we have no control over them (or at least not now) and we don't want to expose these warnings to a user using the Authenticator as a framework

Reply:
Ok, removed.

Reply:
but we don't want to merge it to master before all the warnings are fixed. We don't want to see an unhappy code executor